components/reddit/sources/new-hot-post-on-a-subreddit/new-hot-post-on-a-subreddit.js

```diff
@@ -0,0 +1,184 @@
+const reddit = require("../../reddit.app.js");
+
+module.exports = {
+  key: "new-hot-spot-on-a-subreddit",
+  name: "New hot spot on a subreddit",
+  description:
+    "Emits an event each time a new host post is added to the top 10 a subreddit.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    reddit,
+    subreddit: {
```

Comment:
You can abstract this prop into the app file, as well, and use it here and with the other source. I see you have a different description, but I think it's fine to have the description for the single prop just read:

"The subreddit you'd like to watch"

Reply:
Interesting, didn't have in the radar what props can be abstracted out. I have modified it now.